app/controllers/terms_controller.rb

```diff
@@ -22,8 +22,8 @@ def create
     @vocabulary = find_vocabulary
     combined_id = CombinedId.new(params[:vocabulary_id], term_params[:id])
     term_form = term_form_repository.new(combined_id)
-    term_form.attributes = term_params.except(:id)
-    term_form.set_languages(params)
+    term_form.attributes = vocab_params.except(:id)
```

Comment:
Ok, maybe we need to adjust SimpleForm then. This seems really confusing to have everything in Vocabulary. And I could see that adding other types (Predicate, PersonalName, etc.) even more confusing.